tests/memoryview/numpy_memoryview.pyx

```diff
@@ -36,8 +36,8 @@ def ae(*args):
         if x != args[0]:
             raise AssertionError(args)
 
-def testcase_no_pypy(f):
-    if hasattr(sys, "pypy_version_info"):
+def testcase_no_pypy(f, _is_pypy=hasattr(sys, "pypy_version_info")):
+    if _is_pypy:
```

Comment:
For some reason, this didn't work. The so-decorated tests still got [run](https://travis-ci.org/github/cython/cython/jobs/766997085) on pypy.

Reply:
Takes a while for travis to catch up. That's not the right build.